ui/k-info-panel/switch_info.kytos

```diff
@@ -173,6 +180,34 @@
        request.done(this.msg_state_success)
        request.fail(this.msg_state_failure)
      },
+     bt_request(){
+       var interfaces = {}
+       interfaces["interfaces"] = this.chosen_interface
+       let request = $.ajax({
+                        type: "POST",
+                        url: this.$kytos_server_api + "kytos/of_lldp/v1/interfaces/" + this.chosen_action.toLowerCase(),
+                        async: true,
+                        contentType: "application/json",
+                        data: JSON.stringify(interfaces),
+       });
+       var _this = this
+       request.done(function(data){
+         let notification = {
+           title: 'Interfaces LLDP' + _this.chosen_action + 'd: Success',
```

Comment:
@Alopalao thanks for improving the title, however, it ended up concatenating without a space, so it ended up like this `Interfaces LLDPDisabled: Success`, if you could add a space that'd be great.

Reply:
Notification message fixed on commit [0c3aaac67309e7495cd3d6ba58ef0e6b121d6368](https://github.com/kytos-ng/topology/pull/81/commits/0c3aaac67309e7495cd3d6ba58ef0e6b121d6368)